index.bs

```diff
@@ -613,6 +619,74 @@ Examples for attributes and attribute match lists:
 ```
 </div>
 
+## Namespaces ## {#namespaces}
+
+The [[HTML]] spec embeds [[HTML#svg-0|SVG]] and [[HTML#mathml|MathML]] content
+and supports several [[HTML#attributes-2|namespaced attributes]].
+To support these, the [=configuration object=] supports
+namespaced element and attribute names in the [=attribute match lists=].
+
+The Sanitizer API uses the namespace model and namespace restrictions
+of the [[HTML]] specification, and to support exactly as much namespaced
+content as HTML does. When specifying element names, a set of fixed namespace
+designators can be used to designate elements in the non-default namespaces.
+Namespace designator and element names are seperated by a
+colon (`":"`, U+003A) character. `svg` designates elements in the
+[=SVG namespace=], and `math` designates elements in the [=MathML namespace=].
+All other elements are in the HTML namespace.
+
+<div class="example">
+* `"p"`: The `p` element in the [=HTML namespace=].
+* `"svg:line"`: The `line` element in the [=SVG namespace=].
+* `"math:mfrac"`: The `mfrac` element in the [=MathML namespace=].
+* `"dc:contributor"`: Invalid. This does not designate an element, and
+  will not match anything.
```

Comment:
#140.

Reply:
I didn't see a good, direct reference. I suspect this will require re-writing a bit when upstreaming into HTML.

I did rewrite a bit to have an actual list of namespaces just above.

Reply:
Yeah, fair enough :slightly_smiling_face: